Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/TaskCreateActivity.java

```diff
@@ -2,12 +2,24 @@
 
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import pg.autyzm.friendly_plans.notifications.ToastUserNotifier;
 
 public class TaskCreateActivity extends AppCompatActivity {
 
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.task_management);
-    }
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.task_management);
+  }
+
+  public void eventCreateTask(View view) {
```

Comment:
Ouch - I was thinking about "next" button as the one which saves tasks :D sorry for misleading issue name and no description, but I think we can just move it (showSuccessMessage) as next action after adding task to db in TaksContainerFragment :)
I know it might be a little bit confusing about this Task's saving feature. At final we would like to achieve:
- Save Task before start adding Steps to it (to simplify situation when therapist turn off the manager app on "Add steps" screen)
- Give the user chance to save task and finish without adding any steps

First concept was to use side progress menu, make it clickable and jump into last screen by clicking the "save" button in the menu. But after some further thinking the last screen with just info "everything is ready to save" seems to be unnecessary - just save and leave might be better :)

It's a little bit difficult for me to explain it :D but to summarize - it would be enough to move showSuccessMessage into TaskContainerFragment, after taskValidation and adding it to db - and it would be great :)